### DISCUSSION TITLE
quit filter but not Publisher

### DISCUSSION THREAD

**Discussion by pr-apes** [speedata/publisher#415] [Q&A] [answered]
@pgundlach,

using `os.exit(0)` in a filter quits the filter, as expected.

But it also quits Publisher, which is not what I need in many cases.

Is there any way to quit filter execution and proceed with Publisher?

Many thanks for your help.

**Comment by pgundlach** [comment, marked as answer]
Just return:

```lua


print("Hello")

if true then
    return
end

print("not reached")

```

**Reply by pr-apes** [reply]
Many thanks for your reply.
